src/components/faceVerification/standalone/hooks/useGiveUpDialog.js

```diff
@@ -7,23 +7,29 @@ import { useDialog } from '../../../../lib/dialog/useDialog'
 import GiveUpDialog from '../components/GiveUpDialog'
 import { FVFlowContext } from '../context/FVFlowContext'
 import { requestIdle } from '../../../../lib/utils/system'
+import Config from '../../../../config/config'
+import { openLink } from '../../../../lib/utils/linking'
 import useFVRedirect from './useFVRedirect'
 
 const useGiveUpDialog = (navigation, type) => {
   const { showDialog } = useDialog()
   const fvRedirect = useFVRedirect()
   const { isFVFlow } = useContext(FVFlowContext)
   const { navigate } = navigation
+  const { fvTypeformUrl } = Config
 
   const onReasonChosen = useCallback(
     (reason = undefined) => {
       const data = pickBy({ reason }, negate(isUndefined))
 
-      if (reason && reason !== 'closed') {
+      if (reason !== 'closed') {
         AsyncStorage.removeItem('hasStartedFV')
```

Comment:
this should be always removed

Reply:
We wanted to catch users (upon your own request) who 'started fv, and did not complete'
so up until they give a reason for the 'why', the modal is shown both on navigate back
and on dashboard with the conditions: Has started fv, did not give up a reason in the survey

Reply:
even if reason is Other the hasStartedFV should be removed as the user was shown a survey.
i'm not sure why you programmed Other to show the form, but the survey was shown, so the storage should be removed

Reply:
the reason is 'closed' when closing the survey not giving any reason not other?
other is a link to the typeform if someone wants to give a more elaborate response?

Reply:
So that means it will bug the user until he fills the survey? what's the reason for not clearing it?

Reply:
nothing more then based on your request to catch users who visited the fv page, but did not start and/or complete.
if you meant just wanting to catch users who clicked away the fv-page (and not shown survey)
then it could be cleared always, sure